pkg/manifests/common.go

```diff
@@ -24,7 +26,7 @@ func getOwnerRefs(deploy *appsv1.Deployment) []metav1.OwnerReference {
 	}}
 }
 
-func namespace(conf *config.Config) *corev1.Namespace {
+func Namespace(conf *config.Config) *corev1.Namespace {
```

Comment:
Not sure why Namespace is being exported now?

Reply:
this was because of a change I had made in a different iteration but that's no longer needed - re converting!